src/HPolytope.jl

```diff
@@ -198,40 +198,49 @@ end # quote
 end # function load_polyhedra_hpolytope()
 
 """
-    vertices_list(P::HPolytope{N};
-                  [backend]=default_polyhedra_backend(P, N),
-                  [prunefunc]=removevredundancy!)::Vector{Vector{N}} where
-                  {N<:Real}
+    vertices_list(P::HPolytope{N}; [backend]=nothing)::Vector{Vector{N}} where {N<:Real}
 
 Return the list of vertices of a polytope in constraint representation.
 
 ### Input
 
-- `P`         -- polytope in constraint representation
-- `backend`   -- (optional, default: `default_polyhedra_backend(P, N)`)
-                  the polyhedral computations backend
-- `prunefunc` -- (optional, default: `removevredundancy!`) function to
-                 post-process the output of `vreps`
+- `P`       -- polytope in constraint representation
+- `backend` -- (optional, default: `nothing`) the polyhedral computations backend
 
 ### Output
 
 List of vertices.
 
-### Notes
+### Algorithm
+
+If the polytope is two-dimensional, the polytope is converted to a polygon in
+H-representation and then its `vertices_list` function is used. This ensures
+that, by default, the optimized two-dimensional methods are used.
+
+It is possible to use `Polyhedra` backend in two-dimensions as well; do this
+by passing a given backend, e.g. `backend=CDDLib.Library()`.
 
+If the polytope is not two-dimensional, the concrete polyhedra manipulation
+library `Polyhedra` is used. The actual computation is performed by a given
+backend; for the default backend used in `LazySets` see `default_polyhedra_backend(N)`.
 For further information on the supported backends see
 [Polyhedra's documentation](https://juliapolyhedra.github.io/Polyhedra.jl/).
 """
-function vertices_list(P::HPolytope{N};
-                       backend=default_polyhedra_backend(P, N),
-                       prunefunc=removevredundancy!
```

Comment:
Why remove the `prunefunc` option?

Reply:
I realized that this option is useless, since we got the `backend` option to decide which backend to apply. Moreover, `polyhedron(P; backend=backend)` is a `Polyhedra.Polyhedron` and we don't plan to work on those types here (better make a PR on Polyhedra directly).

Reply:
> I realized that this option is useless

But now you always prune if `Polyhedra` is used and never prune otherwise, which is inconsistent. Maybe we should have a `prune::Bool` argument instead?

Reply:
OK i agree it's a nice idea to add this boolean option to the function. So i'm putting https://github.com/JuliaReach/LazySets.jl/pull/1342/ as a requirement for this PR.